packages/web/src/public-site/pages/landing-page/components/FeaturedContent.tsx

```diff
@@ -133,31 +132,19 @@ type FeaturedContentProps = {
   setRenderPublicSite: (shouldRender: boolean) => void
 }
 
-const getImageUrl = (
-  cid: Nullable<string>,
-  size: SquareSizes,
-  storageNodeSelector: Maybe<StorageNodeSelectorService>,
-  cidMap: Nullable<{ [key: string]: string }> = null
-) => {
-  if (!storageNodeSelector || !cid) return null
-  const node = storageNodeSelector.getNodes(cid)[0]
-  return cidMap && cidMap[size]
-    ? `${node}/content/${cidMap[size]}`
-    : `${node}/content/${cid}/${size}.jpg`
-}
-
 const FeaturedContent = (props: FeaturedContentProps) => {
   const { history } = useHistoryContext()
-  const { storageNodeSelector } = useAppContext()
   const [trendingPlaylistsResponse, fetchTrendingPlaylists] =
     useAsyncFn(async () => {
       const featuredContent = await fetchExploreContent(env.EXPLORE_CONTENT_URL)
       const ids = featuredContent.featuredPlaylists
-      const playlists = audiusBackendInstance.getPlaylists(
-        null,
-        ids
-      ) as any as UserCollectionMetadata[]
-      return playlists
+      const sdk = await audiusSdk()
+      const playlists = await sdk.full.playlists.getBulkPlaylists({
+        id: ids.map((id) => Id.parse(id))
+      })
+      return (playlists.data
```

Comment:
Why not `transformAndCleanList`? Is the boolean thing catching a specific case here?